mlflow/protos/service.proto

```diff
@@ -682,9 +682,21 @@ message ListExperiments {
   // If unspecified, return only active experiments.
   optional ViewType view_type = 1;
 
+  // Maximum number of experiments desired.
+  // Servers may select a desired default value and threshold for `max_results`.
+  // Callers of this endpoint are encouraged to pass max_results explicitly and
+  // leverage page_token to iterate through experiments.
```

Comment:
Can we update the `SearchRuns` documentation for the `max_results` field to read similarly? Currently, it just says `Maximum number of runs desired. Max threshold is 50000`, which may be inaccurate for certain backends (which may support even more results per page).

Reply:
Got it, I'll address this in https://github.com/mlflow/mlflow/pull/4455